### PR TITLE
File structure

### PR DESCRIPTION
**Pull request by Wydder**
The branch LoginSignup is also merged here. 
Top-bar pages:
1. Created About module with container
2. Created InConsultation module with container
3. Created Archive module with container
4. Created Members module with container

### CODE REVIEW

**Comment by Wydder** [comment]
I pushed here the changes made upon review from LoginSignup branch.

File: src/app/core/http/authentication-api.service.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+@Injectable({ providedIn: 'root' })
+export class AuthenticationApiService {
+  private currentUserSubject: BehaviorSubject<User>;
+  public currentUser: Observable<User>;
+
+  constructor(
+    private http: HttpClient,
+    private authenticationService: AuthenticationService
+  ) {
+    this.currentUserSubject = new BehaviorSubject<User>(JSON.parse(localStorage.getItem('currentUser')));
```
Comment (bogdanconstantinescu) [inline src/app/core/http/authentication-api.service.ts]: This bit should also be added to AuthenticationService as `getCurrentUser()`

Comment (Wydder) [reply src/app/core/http/authentication-api.service.ts]: Right, I should have noticed my self. Thank you

**Comment by Wydder** [comment]
I pushed with the last review from Bogdan

File: src/app/core/services/authentication.service.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+@Injectable({ providedIn: 'root' })
+export class AuthenticationService {
+  constructor() { }
+
+  setCurrentUser(user) {
+    localStorage.setItem('currentUser', JSON.stringify(user));
+  }
+
+  getCurrentUser() {
+    localStorage.getItem('currentUser');
```
Comment (bogdanconstantinescu) [inline src/app/core/services/authentication.service.ts]: You missed a `return` :)

**Comment by Wydder** [comment]
I also pushed modules needed for left-menu.

**Pull request merged**